spec/migrations/remap_to_fa6_icon_names_spec.rb

```diff
@@ -12,36 +12,28 @@
       "far fa-ambulance" => "far-truck-medical",
       "fab fa-ambulance" => "fab-truck-medical",
       "fas fa-ambulance" => "truck-medical",
+      "fa-gear" => "gear",
+      "far-gear" => "far-gear",
+      "fab-gear" => "fab-gear",
+      "far fa-gear" => "far-gear",
+      "fab fa-gear" => "fab-gear",
+      "fas fa-gear" => "gear",
+      "gear" => "gear",
```

Comment:
Covers all permutations of prefixes we want to modify; `gear` here is already in FA6.